tkupchyn/homework_08/buildings.py

```diff
@@ -0,0 +1,43 @@
+# Details: https://py.checkio.org/mission/building-base/share/3de02090eb5fac4f0da0ced85f9f3f61/
+
+
+class Building:
+    def __init__(self, south, west, width_we, width_ns, height=10):
+        self.south = south
+        self.west = west
+        self.width_we = width_we
+        self.width_ns = width_ns
+        self.height = height
+
+    def __str__(self):
+        return f"Building({self.south}, {self.west}, {self.width_we}, {self.width_ns}, {self.height})"
+
+    @property
+    def ne_corner(self) -> list:
+        return [self.south + self.width_ns, self.west + self.width_we]
+
+    @property
+    def nw_corner(self) -> list:
+        return [self.south + self.width_ns, self.west]
+
+    @property
+    def se_corner(self) -> list:
+        return [self.south, self.west + self.width_we]
+
+    @property
+    def sw_corner(self) -> list:
+        return [self.south, self.west]
```

Comment:
corner not easy to reuse later, you never know if in corner 1 val is south or west, logic is confusing

but when you have:
self.east
self.west
self.north
self.south

you can prepare any corners you like

Reply:
True:)
For the first time I didn't get what I should change, but then I realized